FHEM/00_SIGNALduino.pm

```diff
@@ -4213,6 +4253,33 @@ attr sduino longids BTHR918N
 <li>rawmsgEvent<br>
 When set to "1" received raw messages triggers events
 </li><br>
+<a name="rfmode"></a>
+<li>rfmode<br>
+Configures the RF transceiver of the SIGNALduino (CC1101). Available arguments are:
+<ul>
+	<li>KOPP_FC<br>
+	Modulation GFSK, DataRate=4.7855 kbps, Sync Word=AA54, Frequency 868.3MHz
+	</li>
+	<li>Lacrosse_mode1<br>
+	Modulation 2-FSK, DataRate=17.25769 kbps, Sync Word=2DD4, Frequency 868.3MHz<br>
+	<ul><small>example: TX25-IT, TX27-IT, TX29-IT, TX29DTH-IT, TX37, 30.3143.IT, 30.3144.IT</small></ul>
+	</li>
+	<li>Lacrosse_mode2<br>
+	Modulation 2-FSK, DataRate=9.579 kbps, Sync Word=2DD4, Frequency 868.3MHz<br>
+	<ul><small>example: TX35TH-IT, TX35DTH-IT, TX38-IT, 30.3155WD, 30.3156WD</small></ul>
+	</li>
+	<li>Lacrosse_mode4<br>
+	Modulation 2-FSK, DataRate=8.842 kbps, Sync Word=2DD4, Frequency 868.3MHz<br>
+	<ul><small>example: TX22 (WS 1600)</small></ul>
+	</li>
+	<li>PCA301<br>
+	Modulation 2-FSK, DataRate=6.62041 kbps, Sync Word=2DD4, Frequency 868.950MHz
+	</li>
+	<li>SlowRF<br>
```

Comment:
Das was zuerst kommt sind irgendwie Hersteller oder Sensorspezifische Dinge, die ja im Prinzip jeder Hersteller so verwenden könnte.

SlowRF sagt mir aber irgendwie nichts. Was ist langsam und für was steht slow?
Wie wäre es mit default als Bezeichnung

Reply:
>SlowRF sagt mir aber irgendwie nichts. Was ist langsam und für was steht slow?
Wie wäre es mit default als Bezeichnung

Schau dazu bitte beim CUL. Die Bezeichnungen wurden gleich übernommen wie auch die Namen der Modes. Wenn wir nun auf einmal andere Namen vergeben, Einspruch. So verwirren wir den User noch mehr. 
"Eine Birne heiß Birne auch wenn ich sie in einem anderen Supermarkt kaufe ;-)" Spaß bei seite. Ich würde gern bei den "FHEM Plattform Bezeichnungen" bleiben wenn sie das gleiche bezwecken. Das ist das selbe Prinzip, das beim CUL das attribut genauso heißt, wie beim SIGNALduino.

Reply:
Gibt es dazu eine Doku im FHEM Universum, denn slow bringe ich mit langsam in Verbindung und die Datenrate ist ja nicht in jedem Fall im Default langsamer als in anderen Fällen. Hängt doch von den Registern ab.

ASK/OOK 433.92 setzen wir mit dem Default oder was machen wir da eigentlich genau in diesem Kontext?

Reply:
Hier gibt es eine Erklärung https://wiki.fhem.de/wiki/SlowRF .

>ASK/OOK 433.92 setzen wir mit dem Default oder was machen wir da eigentlich genau in diesem Kontext?

Ja wir setzen das im default bzw. wenn der User SLowRF einstellt.